### ISSUE TITLE
Seminearring is exported by V5 modules, but never used

### ISSUE THREAD

**Issue by wraithm** [bitemyapp/bloodhound#214]
Seminearring is exported for `Database.Bloodhound.V5.Types`, but it's never actually used. None of the `V5` types have Seminearring instances. That should probably be removed.

**Comment by bitemyapp** [comment]
I agree, it's a weird vestige of an off-hand thought I had a long time ago. Thank you!

**Comment by JoseD92** [comment]
I will take the issue

**Comment by blackheaven** [comment]
ES 5 is not supported anymore.
